notebooks/en/llm_judge.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>Why did we initially have the scale between 0 and 10, and now it's 1 to 4? Shouldn't the range be the same as in the dataset? And could've the difference in scale affected the initial performance? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Since we use pearson correlation to compute the performance, the results should not be affected by a rescaling. But what we show here is that smaller integer scales to work better!</p>